### DISCUSSION TITLE
[script-setup] What's the compiled output when declaring additional props?

### DISCUSSION THREAD

**Discussion by wenfangdu** [vuejs/rfcs#374] [RFC Discussions]
[RFC Link](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0040-script-setup.md#declaring-additional-options)

What's the compiled output of the following code?

```vue
<script>
  export default {
    name: 'CustomName',
    inheritAttrs: false,
    customOptions: {},
  }
</script>

<script setup>
  // script setup logic
</script>
```

**Comment by edison1105** [comment]
is this what you looking for ? see [sfc](https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8aDE+e3sgbXNnIH19PC9oMT5cbjwvdGVtcGxhdGU+XG48c2NyaXB0PlxuICBleHBvcnQgZGVmYXVsdCB7XG4gICAgbmFtZTogJ0N1c3RvbU5hbWUnLFxuICAgIGluaGVyaXRBdHRyczogZmFsc2UsXG4gICAgY3VzdG9tT3B0aW9uczoge30sXG4gIH1cbjwvc2NyaXB0PlxuPHNjcmlwdCBzZXR1cD5cbmNvbnN0IG1zZyA9ICdIZWxsbyBXb3JsZCEnXG48L3NjcmlwdD5cblxuIn0=)

**Reply by wenfangdu** [reply]
@edison1105 Yes, thanks for sharing!
